hase/perf/__init__.py

```diff
@@ -92,6 +95,9 @@ def write(self, directory: str) -> Trace:
             )
             cpus.append(cpu_trace)
 
+        if len(cpus) == 0:
+            l.warning("No cpu traces were recorded")
```

Comment:
@Harvey-Lu if you see this message something went wrong with the recording.

Reply:
This can happen if you record an inactive parent process that is scheduled only when an event happens.